haskell_scotty/Server.hs

```diff
@@ -18,8 +18,8 @@ main = scotty 6969 $ do
 
   post "/api/clients" $ do
     client <- jsonData
-    client <- liftIO $ saveClient client
-    jsonAPI client
+    client' <- liftIO $ saveClient client
```

Comment:
Cool. I felt a little dirty doing this, since it felt like mutation, even though its not. So, `'foo` is convention for indicating a "mutated" copy of `foo`?